src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -238,7 +238,94 @@ public Task Get()
                 }
             }
         }
-        
+
+        [RavenAction("/admin/config-expiration-bundle", "POST", "/admin/config-expiration-bundle?name={databaseName:string}")]
```

Comment:
Per database configuration should be in `/databases/<db-name>`.
Ensures that we can configure this properly and can wait on the db to accept it.